src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Json/SimplePatient-expected.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
To keep the same structure as `HL7v2` and `C-CDA`, how about adding a subfolder for these data, such as `TestData/Expected/Json/Sample/SimplePatient-expected.json`?

Reply:
Sure, subfolders are created according to the root templates they use.